Content.Shared/Storage/StorageComponent.cs

```diff
@@ -20,6 +22,25 @@ public sealed partial class StorageComponent : Component
         [ViewVariables]
         public Container Container = default!;
 
+        /// <summary>
+        /// The max number of entities that can be inserted into this storage.
+        /// </summary>
+        [DataField, ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
+        public int MaxSlots = 7;
+
+        /// <summary>
+        /// The maximum size item that can be inserted into this storage,
+        /// </summary>
+        [DataField, ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
+        [Access(typeof(SharedStorageSystem))]
+        public ItemSize? MaxItemSize;
```

Comment:
Might be safest to just use normal? not sure.

Reply:
a null value is interpreted as "one less than the entity's item size" This will dodge a lot of test fails that could happen easily otherwise.